package.json

```diff
@@ -40,7 +40,8 @@
     "create-modules:native": "node ./scripts/release/createNativeModules.js",
     "create-modules:web": "node ./scripts/release/createWebModules.js",
     "version": "ts-node --project ./scripts/tsconfig.json ./scripts/release/BumpVersion.ts",
-    "validate-staged-widget-versions": "node scripts/validation/validate-versions-staged-files.js"
+    "validate-staged-widget-versions": "node scripts/validation/validate-versions-staged-files.js",
+    "update:widgets-data": "ts-node --project ./scripts/tsconfig.json scripts/data/updateWidgetsData.ts"
```

Comment:
Can we use something more meaningful like `update:widgets-metrics`? It’s very similar to data widgets as Chak said and might cause confusion for people cloning/forking the repo.

Reply:
Yea true. But then again, they're not really metrics. What about `update:widgets-overview`?

Reply:
Resolving this conversation on behalf of vacation Diego as this comment is similar to mine and Kevin has changed it.